http-builder-ng-core/src/main/java/groovyx/net/http/JavaHttpBuilder.java

```diff
@@ -323,6 +323,16 @@ protected Object doDelete(final ChainedHttpConfig requestConfig) {
         return createAndExecute(requestConfig, "DELETE");
     }
 
+    protected Object doPatch(final ChainedHttpConfig requestConfig) {
+        // TODO Fail fast here?
+        // We know that HttpURLConnection.setRequestMethod will throw an exception
+        // for the PATCH method.
+        // A hack-around is to set header "X-HTTP-Method-Override=PATCH" and
+        // send a POST message instead.  It looks like this would be done around
+        // line 58 above.
+        return createAndExecute(requestConfig, "PATCH");
```

Comment:
As discussed in the issue thread, go ahead and throw some sort of `RuntimeException` here. The core PATCH test will just be a standalone test to ensure that the exception is thrown (disconnected from the test kit).

After this gets merged into `development` I will update the User Guide to document that PATCH is not supported in the core implementation and why.